admin/views/admin.php

```diff
@@ -14,11 +14,37 @@
  * @copyright 2016 WisdmLabs or Company Name
  */
 
-?>
+$status  = get_option( 'learndash-gift-edd_license_status', '' );
 
+?>
 <div id="wdm-learndash-gift-edd-setting" class="wrap">
+    <h2><?php echo esc_html( get_admin_page_title() ); ?></h2>
+    <form method="post">
+        <table class="form-table">
+            <tr>
+                <th>
+                    <label for="learndash-gift-edd_license_key"><?php _e( 'License Key', 'learndash-gift-edd' ); ?></label>
+                </th>
+                <td>
+                    <p>
+					    <?php if ( $status == 'valid' ) : ?>
+						    <?php submit_button( 'Deactivate License', 'secondary', 'learndash-gift-edd_license_deactivate', false ); ?>
+                            <span style="color:green">&nbsp;&nbsp;<?php _e( 'active', 'learndash-gift-edd' ); ?></span>
```

Comment:
Maybe just use a left margin instead of `&nbsp;`

Reply:
Whoa. those where being hidden by PHPStorm

Reply:
Fixed